nodejs/awsx/apigateway/api.ts

```diff
@@ -402,43 +402,6 @@ export interface APIArgs {
     deploymentArgs?: DeploymentArgs;
 }
 
-/**
- * Additional optional args that can be passed along to the RestApi created by the
- * awsx.apigateway.API.
- */
-export interface RestApiArgs {
```

Comment:
I actually wasn't suggesting we remove this type completely as that would be a breaking change (since this is an exported type), but I believe exporting this as an alias:

`export type RestApiArgs = aws.apigateway.RestApiArgs;` would be non-breaking since any external references to it should continue to work.

Reply:
Way too sensible! :D Updated ;)